initrd/etc/functions

```diff
@@ -267,6 +267,19 @@ combine_configs() {
 	cat /etc/config* > /tmp/config
 }
 
+# Generate secret value using first 40 chars of ROM SHA256 hash
+secret_from_rom_hash() {
+	local ROM_IMAGE="/tmp/coreboot-notpm.rom"
+
+	echo -e "\nTPM not detected; measuring ROM directly\n" 1>&2
+	# use a previously-copied image if it exists
```

Comment:
@MrChromebox @daringer @jans23 @kylerankin @szszszsz @osresearch @fhvyhjriur @Tonux599 @Thrilleratplay @irelativism @blobless @lrvick

1. Replace this by hardcoded value obtained from backuped flash. (apply same recipe to obtain hash)
2. Do whatever you want inside of ROM secured by Nitrokey Pro/Nitrokey Storage/Librem Key for which "[you can now use a Librem Key in place of your TPM chip and get similar protection against tampering!"](https://web.archive.org/web/20201108000216/https://puri.sm/posts/coreboot-news-new-script-pre-built-binaries-and-pureboot-on-non-tpm-laptops/)
3- Succeed HOTP challenge at every boot until next firmware upgrade.

We merge?